README.md

```diff
@@ -1,139 +1,120 @@
 # My OSX Configuration
 
-## Chrome
-Install Chrome and set it as the default browser
-Add the [Vimium Chrome Extension](https://vimium.github.io/)
+This is my OSX setup for Sierra 10.12. I like a simple setup.
 
 ## Applications to download
-**Spectacle** - For quick setting of your application window sizes
 
-**Karabiner** - Faster key repeat and key remapping
+[**Googe Chrome**](https://github.com/shouston3/My-Config/tree/master#googe-chrome) - My browser choice
```

Comment:
@shouston3 agree that Chrome is go-to browser for several reasons.
but remind me to show you some insanely cool features of https://www.mozilla.org/en-GB/firefox/developer/ 😉